README.md

```diff
@@ -6,13 +6,13 @@ The TOB is responsible for managing conflicts, violations of procedures or guide
 
 * **Brandon Mitchell [Independent]** (term date: 1/29/2023 - 1/29/2025)
 * **Vincent Batts [Microsoft]** (term date: 1/29/2023 - 1/29/2025)
-* **Josh Dolitsky [Chainguard]** (term date: 1/29/2022 - 1/29/2024)
-* **Phil Estes [AWS]** (term date: 1/29/2022 - 1/29/2024)
-* **Jon Johnson [Chainguard]** (term date: 1/29/2022 - 1/29/2024)
-* **Nisha Kumar [Oracle]** (term date: 1/29/2022 - 1/29/2024)
-* **Samuel Karp [Google]** (term date: 1/29/2022 - 1/29/2024) [Chair]
 * **Derek McGowan [Docker]** (term date: 1/29/2023 - 1/29/2025)
 * **Aleksa Sarai [SUSE]** (term date: 1/29/2023 - 1/29/2025)
+* **Phil Estes [AWS]** (term date: 1/29/2024 - 1/29/2026)
+* **Samuel Karp [Google]** (term date: 1/29/2024 - 1/29/2026) [Chair]
```

Comment:
Probably worth removing [Chair] until we have the new chair election.

Reply:
on 2/1 maybe?

Reply:
np.. done :-)